utilities_common/cli.py

```diff
@@ -520,8 +520,8 @@ def run_command(command, display_cmd=False, ignore_error=False, return_cmd=False
     proc = subprocess.Popen(command, shell=True, text=True, stdout=subprocess.PIPE)
 
     if return_cmd:
-        output = proc.communicate()[0]
-        return output
+        command_stdout, command_stderr = proc.communicate()
+        return command_stdout, command_stderr, proc.returncode
```

Comment:
>return [](http://example.com/codeflow?start=8&length=6)

You changed the return type, which will impact other code which uses this function.

Reply:
I updated the code which uses this function to get the standard output of a specific command.

Reply:
Could you find a way not impacting other code? They are not relevant to your feature.

Reply:
Updated!